docker/usr/bin/entrypoint

```diff
@@ -22,6 +22,13 @@ for FOLDER in /data/gitea/conf /data/gitea/log /data/git /data/ssh; do
     mkdir -p ${FOLDER}
 done
 
+if [ -f /data/gitea/conf/app.ini ]; then
+    echo "Found app.ini config file, migrating database"
+    chmod 644 /data/gitea/conf/app.ini
+    chown -R ${USER}:${USER} /data/git /data/gitea
```

Comment:
Heya!

Just remembering that this is still stuck in the queue and I think it's probably a good idea to have this.

To get my LGTM you need to add to the top of file:
```
if [ -z "${USER_GID}" ]; then
  USER_GID="`id -g ${USER}`"
fi
if [ -z "${USER_UID}" ]; then
  USER_UID="`id -u ${USER}`"
fi
```

and change the chown to:

```
chown -R ${USER_UID}:${USER_GID} /data/git /data/gitea
```

Reply:
@zeripath I've just pushed these changes.